packages/cli/src/utils.ts

```diff
@@ -70,16 +72,36 @@ export async function parseConfigFilename(filename: string): Promise<string> {
   return filename;
 }
 
-export function ora(opts?: realOra.Options): realOra.Ora {
-  return realOra({ stream: process.stdout, ...opts });
+export function logSuccess(message: string) {
+  spinner.succeed(message);
+}
+
+export function logFail(message: string) {
+  spinner.fail(message);
+}
+
+export function logStart(message: string) {
+  if (process.stdout.columns == 0 && process.stdout.rows == 0) {
```

Comment:
How about the following to define the logger methods:

```js
class Logger {
  success();
  fail();
  start();
  info();
  warn();
}
class DefaultLogger extends Logger {
  start() { console.log(...) }
}
class TtyLogger extends Logger {
  start() { this.spinner.start(...) }
}
// set the logger by the environment
let logger = new DefaultLogger();
const { columns, rows, isTTY } = process.stdout;
if (isTTY && columns > 0 && rows > 0) {
  logger = new TtyLogger();
}
export logger;
```

Wrapping the TTY mode in class might be more maintainable for extending other methods in the future :)